source/dokku.rst

```diff
@@ -0,0 +1,222 @@
+                   Exposed ports:       -
+                   Id:                  19f9c4618b80ff500d7ea296993e995b423692c1a05da226f483de654ba3c752
+                   Internal ip:         172.17.0.16
+                   Links:               -
+                   Service root:        /var/lib/dokku/services/postgres/{{ project_name }}-database
+                   Status:              running
+                   Version:             postgres:9.6.1
+    $ ssh dokku postgres:link {{ project_name }}-database {{ project_name }}
+            no config vars for test_template
+            -----> Setting config vars
+                   DATABASE_URL: postgres://postgres:5a7dcf4d92435f287a582de383fc99da@dokku-postgres-test-template-database:5432/test_template_database
+            -----> Restarting app test_template
+            App test_template has not been deployed
+
+Set a secret key::
+
+    $ ssh dokku config:set {{ project_name }} DJANGO_SECRET_KEY=$(make generate-secret)
+            -----> Setting config vars
+                   DJANGO_SECRET_KEY: cHDFoeujP9oJkODdxPrXICWM0jBDKyVz
+            -----> Restarting app test_template
+            App test_template has not been deployed
+
+Set an environment name::
+
+    $ ssh dokku config:set {{ project_name }} ENVIRONMENT=production
+    -----> Setting config vars
+           ENVIRONMENT: production
+    -----> Restarting app test_template
+    App test_template has not been deployed
+
+Set the domain name::
+
+    $ ssh dokku config:set {{ project_name }} DOMAIN={{ project_name }}.$(ssh dokku domains:report {{ project_name }} --domains-global-vhosts)
+    -----> Setting config vars
+           DOMAIN: test_template.caktustest.net
+    -----> Restarting app test_template
+    App test_template has not been deployed
+
+Deploy::
+
+    $ git remote add dokku dokku:{{ project_name }}
+    $ git push dokku master
+    Counting objects: 82, done.
+    Delta compression using up to 4 threads.
+    Compressing objects: 100% (73/73), done.
+    Writing objects: 100% (82/82), 89.87 KiB | 5.99 MiB/s, done.
+    Total 82 (delta 2), reused 0 (delta 0)
+    -----> Cleaning up...
+    -----> Building test_template from herokuish...
+    -----> Adding BUILD_ENV to build environment...
+    -----> Warning: Multiple default buildpacks reported the ability to handle this app. The first buildpack in the list below will be used.
+           Detected buildpacks: multi nodejs python
+    -----> Multipack app detected
+    =====> Downloading Buildpack: https://github.com/heroku/heroku-buildpack-nodejs.git
+    =====> Detected Framework: Node.js
+
+    -----> Creating runtime environment
+
+           NPM_CONFIG_LOGLEVEL=error
+           NPM_CONFIG_PRODUCTION=true
+           NODE_VERBOSE=false
+           NODE_ENV=production
+           NODE_MODULES_CACHE=true
+
+    -----> Installing binaries
+           engines.node (package.json):  >=4.2 <4.3
```

Comment:
It appears that since this work was done, the current versions of NPM no longer support 4.2 and since NodeJS 6.x is currently LTS, maybe this should include an update to both address that issue and update the DPT at the same time.

Reply:
Can you make a pull request to update node/npm in the DPT and make sure it all still works?    Then I can update these pull requests to update the example output, if that seems worthwhile to you.

Reply:
I will do that. It might take a little time, just because of the tediousness of testing DPT changes. I'll make that my priority tomorrow.

Reply:
@dpoirier Here's the PR to update node, as well as psycopg2 because of another error: https://github.com/caktus/django-project-template/pull/287